deploy/packages/bbp-packages.yaml

```diff
@@ -30,7 +30,7 @@ packages:
       - brainbuilder@0.11.7
       - brion@3.0.0 +python ^boost@1.65.0
       - connectome-tools@0.3.2
-      - spatial-index@0.1.0
+      - spatial-index@0.2.1
```

Comment:
Is this part of the simulation stack? Wasn't aware of that before…

Reply:
No, just to avoid opening a separate PR